challenge_open/src/challenge_open.py

```diff
@@ -44,26 +40,6 @@ def execute(self, userdata):
         return "done"
 
 
-class StartDetectingSkeletonsForever(smach.State):
```

Comment:
Would it make sense to put this in robot_smach_states, in a new section for gestures etc.